Source/SeaInk.Core/Entities/Tables/ExcelTable.cs

```diff
@@ -0,0 +1,209 @@
+﻿using System.Collections.Generic;
+using System.IO;
+using ClosedXML.Excel;
+using SeaInk.Core.Models.Tables;
+
+namespace SeaInk.Core.Entities.Tables
+{
+    public class ExcelTable : ITable
+    {
+        private XLWorkbook _workbook = null!;
+        private string _pathToFile;
+
+
+        public int SheetCount => _workbook.Worksheets.Count;
+        
+        public int ColumnCount(TableIndex index)
+        {
+            return 0;
+        }
+
+        public int RowCount(TableIndex index)
+        {
+            return 0;
+        }
+
+        public void CreateSheet(TableIndex index)
+        {
+            _workbook.Worksheets.Add(index.SheetName);
+            Save();
+        }
+
+        public void DeleteSheet(TableIndex sheet)
+        {
+            _workbook.Worksheets.Delete(sheet.SheetName);
+
+            var oldPath = _pathToFile;
+            _pathToFile = SeparatePath(_pathToFile).path;
+            _pathToFile += "t.xlsx";
+            
+            Save();
+            File.Delete(oldPath);
+            
+            Rename(SeparatePath(oldPath).name);
+
+        }
+
+        public void Load(string address)
```

Comment:
Можешь себе позволить
```suggestion
        public void Load(string path)
```